### PR TITLE
Add 'Naming resources in Databricks' page

### PR DESCRIPTION
**Pull request by sake92**

### CODE REVIEW

**Comment by sake92** [comment]
CC @lkrystkowiak-snowplow

File: docs/getting-started-with-snowplow-bdp/cloud/naming-resources-in-databricks/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+We advise to use the default resource names in the setup script provided in Console. If you need to follow your own naming convention however, please follow the guidelines below and provide Snowplow with the resource names you used through the Databricks setup in BDP Console.
+
+:::
+
+### `HTTP path`
+
+[Databricks compute resources URL](https://docs.databricks.com/integrations/jdbc-odbc-bi.html#building-the-connection-url-for-the-databricks-driver).
+
+It should start with a slash character.
```
Comment (lkrystkowiak-snowplow) [inline docs/getting-started-with-snowplow-bdp/cloud/naming-resources-in-databricks/index.md:18]: this is fixed now, we do accept HTTP paths not starting with `/`

File: docs/getting-started-with-snowplow-bdp/cloud/naming-resources-in-databricks/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+For more details, see [Configure the Databricks JDBC drivers](https://docs.databricks.com/integrations/jdbc-odbc-bi.html).
+
+### `Schema name`
+
+Default name: `snowplow`
+
+[Specifies the identifier for the schema](https://docs.databricks.com/sql/language-manual/sql-ref-names.html#schema-name); must be unique for the database in which the schema is created.
+
+The schema identifier needs to follow these rules:
+- begin with an letter, number or or underscore character
```
Comment (lkrystkowiak-snowplow) [inline docs/getting-started-with-snowplow-bdp/cloud/naming-resources-in-databricks/index.md:29]: * an => a
* remove extra "or"

File: docs/getting-started-with-snowplow-bdp/cloud/naming-resources-in-databricks/index.md (hunk 1 above)
Comment (sake92) [reply docs/getting-started-with-snowplow-bdp/cloud/naming-resources-in-databricks/index.md]: Fixed, thx!

**Comment by stanch** [comment]
Superseded by #325

**Pull request closed without merging**